nan_callbacks.h

```diff
@@ -101,7 +101,7 @@ static const int kIndexPropertyQueryIndex =      5;
 static const int kIndexPropertyFieldCount =      6;
 
 template<typename T, typename P>
-P *GetWrapper(T needle) {
+static inline P *GetWrapper(T needle) {
```

Comment:
Just `inline` is sufficient but each compilation unit will now end up with its own instance of `haystack`.  I don't think that's the intended behavior.

To be honest, I don't really understand why `haystack` exists.  Can't it just be removed?